modules/data.land/R/soilgrids_texture_extract.R

```diff
@@ -0,0 +1,129 @@
+##' soilgrids_texture_extraction function
+##' A function to extract and save three types of soil texture data in parallel for a single or group of 
+##' lat/long locations based on user-defined site location from SoilGrids250m 
+##' version 2.0 : https://soilgrids.org
+##' @title soilgrids_texture_extraction
+##' @name soilgrids_texture_extraction
+##' 
+##' @param site_info A data frame of site info containing the BETYdb site ID, 
+##' site name, latitude, and longitude, e.g. 
+##' (site_id, site_name, lat, lon)
```

Comment:
The `site_name` is not necessary for setting up the site lists (see my below comments).

Reply:
Fixed.